docs/NewProject.md

```diff
@@ -0,0 +1,44 @@
+﻿<!-- Copyright (c) Microsoft Corporation. All rights reserved.
+     Licensed under the MIT License. -->
+     
+## Adding a new project
+Do the following when adding a new project:
+
+### For all projects
+1. Add a project to the Axe.Windows Solution (`src\axe.windows.sln`).
+2. Right-click on the project and select Properties.
+2. In the Application tab, configure "Target Framework" to use the same .NET Framework version used by the `axe.windows.core` project.
+   - Currently .NET Framework 4.7.1 is the target. 
+3. In the build tab, set the following for both Debug and Release configurations:
+   1. "Warning level" to 4.
+   2. "Treat warnings as errors" to "All".
+
+### For *non-test* projects only
```

Comment:
>*non-test* [](start = 8, length = 10)

Once we remove signing, the "non-test projects only" section will go away and the "test projects only" simply become part of the "all projects" section. Settings.targets should also go away. Not expecting this to be part of this PR, but just capturing this for the later change.

Reply:
I'm wondering if we should leave this until we decide we definitely will not support signing any time in the near future. This decision will be made soon.